README.md

```diff
@@ -15,15 +15,21 @@ _We use `verify` instead of `package` because there are tests in the verify stag
 
 _Hint: If the build fails, it may be because a package in the Docker image has been recently updated. To work around this, see the [Working with Pinned OS Packages](https://github.com/uclalibrary/docker-cantaloupe#working-with-pinned-os-packages) section at the bottom of this document._
 
-To build the latest nightly build of Cantaloupe, use the following:
+To build Cantaloupe using the latest code on [the `develop` branch](https://github.com/cantaloupe-project/cantaloupe/tree/develop) (i.e., our "nightly" build), use the following:
```

Comment:
Maybe we should say on "the upstream `develop` branch" to make it clear this isn't a Cantaloupe fork? I've fielded questions from third parties that were really about the Cantaloupe code (not our packaging of the build artifact).

Reply:
Though if they highlight the link they'll see the source 🤷🏼

Reply:
I think it's good to clarify